packages/trader/src/AppV2/Containers/Notifications/notifications.tsx

```diff
@@ -1,14 +1,25 @@
-import { useNotifications, NotificationBanners } from '@deriv-com/quill-ui';
 import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { useNotifications, NotificationBanners } from '@deriv-com/quill-ui';
+import { StandaloneFlagCheckeredFillIcon } from '@deriv/quill-icons';
+
+const Notifications = observer(() => {
+    const { addBanner, banners, removeBanner } = useNotifications();
+    const { portfolio } = useStore();
+    const { setAddNotificationBannerCallback } = portfolio;
+
+    React.useEffect(() => {
+        const addNotificationBannerCallback = (params: Parameters<typeof addBanner>[0], result: string) =>
+            addBanner({
+                icon: <StandaloneFlagCheckeredFillIcon iconSize='sm' className={`trade-notification--${result}`} />,
+                ...params,
+            });
 
-const Notifications = () => {
-    const { banners, removeBanner } = useNotifications();
+        setAddNotificationBannerCallback(addNotificationBannerCallback);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
 
-    return (
-        <>
-            <NotificationBanners banners={banners} onClose={removeBanner} />
-        </>
-    );
-};
+    return <NotificationBanners banners={banners} isMobile onClose={removeBanner} className='trade-notification' />;
```

Comment:
is it fine if we provide `autohideTimeout`  if 4 seconds is fixed for our dtrader app, but if we are sure quill will never change the default value then we can ignore

Reply:
I agree, better to pass 4 sec, who knows when they'll change it. Will apply suggestion, thank you